indigo_app/templatetags/indigo_app.py

```diff
@@ -30,6 +30,14 @@ def user_profile(user):
     return html
 
 
+@register.simple_tag
+def user_profile_photo_thumbnail(user, height=32, width=32):
+    if not user:
+        return ''
+
+    return format_html('<img src="{}" height="{}" width="{}" style="border-radius: 50%;">'.format(user.userprofile.profile_photo_url, height, width))
```

Comment:
```suggestion
    return format_html('<img src="{}" height="{}" width="{}" class="user-profile-photo">'.format(user.userprofile.profile_photo_url, height, width))
```

Then add a class in `_core.scss` that gives it a 5px radius, so we're consistent everywhere.